web/modules/custom/dgreat_modal_webforms/src/Plugin/Block/DgreatModalSurveyForm.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\dgreat_modal_webforms\Plugin\Block;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountInterface;
+Use Drupal\user\Entity\User;
+
+/**
+ * Provides a 'DgreatModalContactForm' block.
+ *
+ * @Block(
+ *  id = "dgreat_modal_student_survey",
+ *  admin_label = @Translation("Student Survey"),
+ * )
+ */
+class DgreatModalSurveyForm extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $my_form = \Drupal\webform\Entity\Webform::load('contact');
```

Comment:
We'll want to change this from `contact` to @dustinleblanc 's survey webform machine name.